### ISSUE TITLE
Got a error when use `chalk.bgHex()` with arg `--no-color`

### ISSUE THREAD

**Issue by gucong3000** [chalk/chalk#176]
```js
const chalk = require('chalk');
try {
	chalk.bgHex('#cccccc')('message');
} catch (ex) {
	console.error(ex);
}
```
```bash
E:\work\gulp-reporter (master) (gulp-reporter@2.1.0)
$ node -v
v8.1.2

E:\work\gulp-reporter (master) (gulp-reporter@2.1.0)
$ node test.js --no-color
TypeError: Cannot read property 'hex' of undefined
    at Function.<anonymous> (E:\work\gulp-reporter\node_modules\chalk\index.js:98:57)
    at errors.map (E:\work\gulp-reporter\lib\formater.js:240:19)
    at Array.map (native)
    at formater (E:\work\gulp-reporter\lib\formater.js:235:18)
    at reporter (E:\work\gulp-reporter\lib\reporter.js:20:10)
    at getErrors.then.errors (E:\work\gulp-reporter\lib\gulp-plugin.js:19:19)

```

**Comment by Qix-** [comment]
duplicate of #175

**Comment by AdrienHorgnies** [comment]
I think I replicated this issue in another manner :

Given the following file :
```javascript
const chalk = require('chalk');


const colors = {
    success: chalk.hex('#06F90B'),
    failure: chalk.hex('#F9060B')
};

const messages = {
    greeting: `${chalk.bgCyan.black('/ᐠ｡ꞈ｡ᐟ\\')} ${chalk.bold(`Oh hai. I'm Node-db-importer v${packageInfo.version}.
I need information before importing your db.`)}`,
    connectionSuccess: `${colors.success('connected to the database')}`,
    connectionFailure: `${colors.failure('failed to connect to the database')}`
};
```

I get the following error message when my IDE try to run the file (say by debugging the file) but not when simply using it (`node thisfile`) : 
```javascript
/usr/bin/node /home/altissiadev/JHipster/db-importer/node_modules/mocha/bin/_mocha --ui bdd --reporter /home/altissiadev/.IntelliJIdea2017.2/config/plugins/NodeJS/js/mocha-intellij/lib/mochaIntellijReporter.js /home/altissiadev/JHipster/db-importer/test/test-constants.js
/home/altissiadev/JHipster/db-importer/node_modules/chalk/index.js:75
				const open = ansiStyles.color[levelMapping[level]][model].apply(null, arguments);
				                                                  ^

TypeError: Cannot read property 'hex' of undefined
    at Chalk.<anonymous> (/home/altissiadev/JHipster/db-importer/node_modules/chalk/index.js:75:55)
    at Object.<anonymous> (/home/altissiadev/JHipster/db-importer/constants.js:61:20)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/altissiadev/JHipster/db-importer/test/test-constants.js:4:13)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /home/altissiadev/JHipster/db-importer/node_modules/mocha/lib/mocha.js:230:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/altissiadev/JHipster/db-importer/node_modules/mocha/lib/mocha.js:227:14)
    at Mocha.run (/home/altissiadev/JHipster/db-importer/node_modules/mocha/lib/mocha.js:495:10)
    at Object.<anonymous> (/home/altissiadev/JHipster/db-importer/node_modules/mocha/bin/_mocha:469:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3

Process finished with exit code 1
```

This is most probably because it uses an option such as `--no-color`.

**Comment by Qix-** [comment]
@AdrienHorgnies See the duplicate I mentioned earlier - #175.

This is fixed by #191.
